receiver/statsdreceiver/README.md

```diff
@@ -12,27 +12,46 @@ The following settings are required:
 
 - `endpoint` (default = `localhost:8125`): Address and port to listen on.
 
+
+The Following settings are optional:
+
+- `aggregation_interval: 70s`(default value is 60s): The aggregation time that the receiver aggregates the metrics
```

Comment:
May be note that this is equivalent/similar to the flush interval in statsd server?